src/middlewared/middlewared/plugins/vm.py

```diff
@@ -297,7 +297,7 @@ def construct_xml(self):
                 'features', attribute_dict={
                     'children': [
                         create_element('acpi'),
-                        create_element('apic'),
```

Comment:
Curious why removing apic?

Reply:
Because it translates to `-I` flag which is non-existent in `bhyve`